src/api/test/test_consistency_helper.rb

```diff
@@ -35,6 +35,9 @@ def resubmit_all_fixtures
     assert_response :success
     packages = Xmlhash.parse(@response.body)
     packages.elements('entry') do |p|
+      # skip multibuild flavors
+      next if p['name'].include?(':')
```

Comment:
Try to use functional approaches when possible. It moves out the complexity from inside each iteration:

```suggestion
    packages.elements('entry').reject {|p| p['name'].include?(':') }.each  do |p|
```